src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -941,6 +941,10 @@
     <!-- Needed by Xaml Task Factory -->
     <Reference Include="System.Xaml" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.InteropServices" />
```

Comment:
Should these be here, or in the `.NETFramework`-only group above with the other bare `System.*` references?

Reply:
Sorry, I've been shotgunning attempts to fix the issue.  I put these back in the conditional.